docs/tutorial/time-server/0/package.json

```diff
@@ -4,5 +4,10 @@
   "version": "1.0.0",
   "maintainer": "https://github.com/mesosphere/universe",
   "description": "This is a simple python http server that displays a webpage which says the current time at the server location.",
-  "tags": ["http", "time-server", "basichttp"]
-}
+  "tags": [
+    "http",
+    "time-server",
+    "basichttp"
+  ],
+  "lastUpdated": 1499380201000
```

Comment:
How did you calculate this value? This is not unix epoch.

Reply:
FYI I have used https://github.com/mesosphere/universe/pull/2398/files#diff-c4ca3bb11e5c023a1b98e58d2c58dcf6L293 to calculate the epoch from commit ref

Reply:
It is the Unix time epoch in milliseconds. On running `git log --format=%aD docs/tutorial/time-server/0/package.json` we get _Thu, 6 Jul 2017 15:30:01 -0700_. On converting this timestamp to actual date we get _Thursday, July 6, 2017 10:30:01 PM_ in UTC which is equal to above time.

Reply:
Okay, please change this value to seconds.

Reply:
Alright

Reply:
Done